src/components/TmLiGovTransaction/index.js

```diff
@@ -0,0 +1,5 @@
+import TmLiGovTransaction from "./TmLiGovTransaction.vue"
```

Comment:
What's the purpose of this file?

Reply:
nothing really, there are a bunch of them so we'll have to delete them all

Reply:
I deleted only the gov one on this PR